react/FileInput/index.jsx

```diff
@@ -0,0 +1,70 @@
+import cx from 'classnames'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.styl'
+
+const FileInput = ({
+  children,
+  className,
+  disabled,
+  hidden,
+  multiple,
+  onChange,
+  ...inputProps
+}) => {
+  // generate a random id for the input
+  const fileinputId = `file-input-${Math.random()
```

Comment:
I don't know if it's best or not but FYI I've used `uniqueId` from lodash to get a unique ID in several components such as [Modal](https://github.com/cozy/cozy-ui/blob/master/react/Modal/index.jsx#L264)

Reply:
Oh cool, I searched for it :-)